packages/theme/pages/MyAccount.vue

```diff
@@ -16,14 +16,13 @@
           <MyProfile />
         </SfContentPage>
 
-        <SfContentPage title="Shipping details">
-          <ShippingDetails />
+        <SfContentPage title="Saved addresses">
+          <SavedAddressesDetails />
         </SfContentPage>
 
-        <SfContentPage title="Billing details">
-          <BillingDetails />
+        <SfContentPage title="My newsletter">
+          <MyNewsletter />
```

Comment:
I guess `MyNewsletter` should be removed too (https://github.com/vuestorefront/spree/issues/146)

Reply:
Fixed.